### ISSUE TITLE
✨ Enhancement: Page in overview section to show Case Studies

### ISSUE THREAD

**Issue by benjagm** [json-schema-org/website#440]
### Is your feature request related to a problem? Please describe

No

### Describe the solution you'd like

it would be amazing to have a new page inside the overview section to show the case studies published in the blog.
This will be a data drive page reading results from a json file stored in our data folder. Each item of the file will have these fields:
- logo
- summary
- List of links
  - lang : Language in ISO 639-1
  - url : link to the case study in that language
  
Please see this prototype:
<img width="542" alt="Screenshot 2024-03-04 at 15 45 26" src="https://github.com/json-schema-org/website/assets/40007659/ca814769-a895-46a3-8558-3893172e5601">

This is a prototype made by screenshot. Please complete the job using styles as per our site.

For the cards please consider using border shadows similar to the ones in  the landing page:
<img width="1019" alt="Screenshot 2024-03-04 at 15 50 28" src="https://github.com/json-schema-org/website/assets/40007659/dfde3d13-750c-4424-b3e9-c0d60aa50c51">

The case studies are available in the blog:
https://json-schema.org/blog?type=Case%20Study

### Are you working on this?

No

**Comment by TamannaVerma99** [comment]
Hey!
Since my previous issue is blocked by another one (the card component is yet to be made) , Is it ok if I can be assigned this issue?

**Comment by benjagm** [comment]
Please go ahead. These cards is not going to be the standard ones. So no blockers expected here.

**Comment by TamannaVerma99** [comment]
@benjagm, Had a few questions related to this one.

In order to add a custom card component which will read data from the specified JSON file, I would either have to use a .tsx file or a .mdx file. However, the present configuration for the pages/SideBar section specifies the use of only '.md' files.  I tried finding out ways to implement components in .md itself, but conversion to .mdx is mandatory and based on the settings in [slug].page.tsx file (specifically, `getStaticMarkdownPaths` property) allows only .md files to be rendered. 

So, will it be okay to modify this function to include .mdx files or is there another way round that can be used?

**Comment by benjagm** [comment]
> I would either have to use a .tsx

Please see the [slug].page.tsx in each subfolder of pages. This can help you think about how you can have your own page without md. I will recommend you to take a look to [implementers page](https://github.com/json-schema-org/website/blob/main/pages/implementations/index.page.tsx) but be careful, this page doesn't have sidebar.

I hope this helps. We have multiple issues implementing similar features. Pleas see #429

**Comment by TamannaVerma99** [comment]
Okay I will take reference from these issues! Thanks for the information.

**Comment by benjagm** [comment]
Can you please make sure the target branch of this PR is : https://github.com/json-schema-org/website/tree/web-release-3?

Reference docs: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request

**Comment by TamannaVerma99** [comment]
> Can you please make sure the target branch of this PR is : https://github.com/json-schema-org/website/tree/web-release-3?
> 
> Reference docs: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request

Done from my side :)

**Comment by TamannaVerma99** [comment]
Hey @benjagm 
Just an update I am done with all the changes you gave on my previous pr ,I will raise another pr as soon as the card component will be merged.
Also I wanna know if there is any content that I have to write at the top of cards on casestudies page.
Thanks :)

**Comment by benjagm** [comment]
> Also I wanna know if there is any content that I have to write at the top of cards on casestudies page.

Make sure there is a title and intro paragraph and it will be enough. It can be "Loren impsum..." because I will be sending a PR with the texts before merging.

**Comment by TamannaVerma99** [comment]
okay thanks got it!

**Comment by benjagm** [comment]
@TamannaVerma99 please proceed with the integration if the component Card available in the parent branch. If you need to modify the component let's discuss first because others are integrating it.

**Comment by benjagm** [comment]
This is already merged to in release-3 branch. Will be closed as soon as that goes to main branch.

**Comment by benjagm** [comment]
Closed as completed.
